mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -433,6 +433,14 @@ def process_code_coverage(key, value, timestamp):
                         self.logger.prn_inf("%s received"% (key))
                         callbacks__exit_event_queue = True
                         break
+                    elif key == '__timeout_set':
+                        # Dynamic timeout set
+                        timeout_duration = int(value) # New timeout
+                        self.logger.prn_inf("setting timeout to: %d sec"% int(value))
+                    elif key == '__timeout_adj':
```

Comment:
I tend to stay away from abbreviations if the full word is reasonably small. In this case `__timeout_adjust` is just three more characters. Would you mind "adjusting" (heh, couldn't resist 😄) this key?

Reply:
Sure.